contracts/TransparentForwarder.sol

```diff
@@ -3,48 +3,26 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
-
-interface IStaking {
-    function isWhitelisted(address caller) external payable returns (bool);
-}
+import "./interface/IStaking.sol";
 
 contract TransparentForwarder is AccessControlEnumerable {
-    bytes32 public constant TRANSPARENT_FORWARDER_ADMIN_ROLE =
-        keccak256("TRANSPARENT_FORWARDER_ADMIN_ROLE");
+    bytes32 public constant TRANSPARENT_FORWARDER_ADMIN_ROLE = keccak256("TRANSPARENT_FORWARDER_ADMIN_ROLE");
     address public forwarder;
     IStaking public staking;
 
     constructor(address _forwarder) {
         _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        // slither-disable-next-line missing-zero-check
         forwarder = _forwarder;
```

Comment:
address  check should be done. Don't ignore slither warnings without reason. If you are ignoring, add a comment why it is ignored.